packages/devtools_app_shared/lib/src/service/eval_on_dart_library.dart

```diff
@@ -29,6 +30,7 @@ class Disposable {
   }
 }
 
+@sealed
```

Comment:
Avoid the legacy annotations that are from before Dart supported proper class modifiers. If this was to be sealed, you would use the `sealed` keyword. In practice I would just avoid making this `sealed` although you might consider making it `final`.
https://dart.dev/language/class-modifiers

Reply:
Discussed offline. `@sealed` seems like the cleanest solution to protect users from breaking changes (by warning then when they extend or implement a class they shouldn't), while not doing something hacky like including the test fake in the library with the final class it implements.

Reply:
https://github.com/flutter/devtools/issues/6239